python-sdk/src/astro/files/locations/base.py

```diff
@@ -7,9 +7,19 @@
 from urllib.parse import urlparse
 
 import smart_open
+from airflow.hooks.base import BaseHook
 
 from astro.constants import FileLocation
 
+CUSTOM_CONN_TYPE_TO_FILE_SCHEME = {
+    "bigquery": {"gs"},
+    "google_cloud_platform": {"gs", "gdrive", "https"},
+    "s3": {"s3"},
+    "aws": {"s3"},
+    "wasb": {"wasb"},
+    "gcpbigquery": {"gs"},
+}
+
```

Comment:
This should instead go in the classes themselves.

```python
class GCSLocation:
     supported_conn_type = {GCSHook.conn_type}

```

Reply:
Changed the logic: https://github.com/astronomer/astro-sdk/pull/1478/commits/181366c944d65117e7ab0d8b06b72ffc9c23a270